code/modules/reagents/Chemistry-Holder.dm

```diff
@@ -235,20 +235,14 @@
 	return amount
 
 /datum/reagents/proc/trans_to_holder(var/datum/reagents/target, var/amount = 1, var/multiplier = 1, var/copy = 0) // Transfers [amount] reagents from [src] to [target], multiplying them by [multiplier]. Returns actual amount removed from [src] (not amount transferred to [target]).
-
 	if(!target || !istype(target))
 		return 0
-
 	if (amount <= 0 || multiplier <= 0)
 		return 0
-
 	amount = max(0, min(amount, total_volume, REAGENTS_FREE_SPACE(target) / multiplier))
-
 	if(!amount)
 		return 0
-
```

Comment:
Like these ones. These kinds of newlines are good, because they segment out parts of a proc. We don't always want everything to be continuous.